app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -76,3 +79,9 @@ Some courses begin with time spent in a university or school where you start you
 There are two types of assessments you’ll have as part of your training - formative and summative. Formative assessments are less formal and happen throughout your course. Summative assessments are formal assessments, usually at the end of your course.
 
 You might be assessed through written assignments, lesson observations, or by a portfolio of evidence showing how you’re meeting the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards). Assessments vary by provider and the qualifications you’re working towards.
+
+## Adjustments and accessibility
+
+Providers can make adjustments if you need supoprt because you're neurodiverse, have a long-term physical or mental health condition, or have any other accessibility needs.
```

Comment:
```suggestion
Providers can make adjustments if you need support because you're neurodiverse, have a long-term physical or mental health condition, or have any other accessibility needs.
```